paasta_tools/utils.py

```diff
@@ -394,6 +394,11 @@ def get_net(self):
         """
         return self.config_dict.get('net', 'bridge')
 
+    def get_volumes(self, system_volumes):
+        volumes = system_volumes + self.get_extra_volumes()
+        deduped = {v['containerPath'] + v['hostPath']: v for v in volumes}.values()
```

Comment:
This seems pretty good. Although it doesn't dictate which config "wins".

One possible problem: say system config has `/thing` mounted to `/thing` and the service config has`/another` mounted to `/thing/sub` then we know that definitely wont work right? Can we catch that case to?